Week1/h.w_select.js

```diff
@@ -0,0 +1,65 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world'
+});
+
+connection.connect();
+
+connection.query('SELECT country_name, population FROM countries where population > 8000000', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('countries with population > 8 millions:\n ', results, '\n');
+});
+
+connection.query('SELECT country_name FROM countries where country_name like "%land%"', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('countries contains "land" in their names:\n ', results, '\n');
+});
+
+connection.query('SELECT city_name, population FROM cities where population between 500000 and 1000000', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('cities with population in between 500000 and 1000000:\n ', results, '\n');
+});
+
+connection.query('SELECT country_name,continent FROM countries where continent = "Europe"', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('countries in the continent Europe:\n ', results, '\n');
+});
+
+connection.query('SELECT country_name,surface_area FROM countries order by surface_area desc ', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('countries ordered descending by their areas:\n ', results, '\n');
+});
+
+connection.query('SELECT country_name,surface_area FROM countries order by surface_area desc ', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('countries ordered descending by their areas:\n ', results, '\n');
+});
+
+connection.query('SELECT city_name  FROM cities where country_name like"Netherlands"', function (error, results, fields) {
+  if (error) throw error.message;
+  console.log('cities in the netherlands:\n ', results, '\n');
+});
+
+connection.query('SELECT city_name, population  FROM cities where city_name like"Rotterdam"', function (error, results, fields) {
```

Comment:
Please notice the difference between like and = before using like. In this case, it does not matter, however there are cases where "like" and "=" give different results